### ISSUE TITLE
Segmentation fault in xmpp_message_archive_management_NS_VER

### ISSUE THREAD

**Issue by joudinet** [dino/dino#1571]
Hi,
I've just installed the package dino-im on a fresh installation of Debian stable (bookworm), using Gnome. After configuring my XMPP account, it immediately crashes. Then, I run it with gdb to identify where it crashes and it reports:
```
Thread 1 "dino-im" received signal SIGSEGV, Segmentation fault.
0x00007ffff7e2170a in xmpp_message_archive_management_NS_VER ()
    from /usr/lib/x86_64-linux-gnu/dino-im/libxmpp-vala.so.0
```
Looking at the backtrace, it was just before in `xmpp_message_archive_management_create_base_query`

I guess this might be due to loading a MUC room (which my account should do automatically when I start my XMPP client).

I use dino-im version 0.4.2-1 according to `dpkg -l`.

Any idea how to circumvent this crash?

**Comment by matthewrfennell** [comment]
I also run 0.4.2-1 on Bookworm. Unfortunately, I haven't been able to replicate this yet. 

However, I'd like to help if I can. Some things off the top of my head that could help narrow it down:

* Do the logs show anything useful?
* Do you have a list of MUCs you're willing to share?
* If you create a dummy account with no MUCs, does it work? Now try adding back each one one by one.
* Does this also happen on the latest version from git?
* Are you able to replicate on another machine / VM / environment?

**Comment by mar-v-in** [comment]
This is a duplicate of #1405 which was fixed in release 0.4.3.

On bookworm, you can install the fixed version from backports.
